app/src/main/java/com/example/bookwormadventuresdeluxe2/Utilities/Status.java

```diff
@@ -4,9 +4,10 @@
  * Ensures that the status of a book is one of these four
  * values
  */
-public enum Status {
-    AVAILABLE,
-    BORROWED,
-    REQUESTED,
-    ACCEPTED
-}
\ No newline at end of file
+public enum Status
+{
+    Available,

```

Comment:
Is there a reason these are no longer capitalized? I think it makes sense to have enums capitalized?

Reply:
It was so I could use status.toString() in the xml